--- origsrc/WordNet-3.0/src/wnb	2006-12-18 23:38:20.000000000 +0100
+++ src/WordNet-3.0/src/wnb	2011-11-07 22:28:21.984375000 +0100
@@ -13,9 +13,9 @@ exec true
 #                                                                        # 
 ##########################################################################
 
-if {$tcl_platform(platform) == "windows"} {
-  package require registry 1.1
-}
+#if {$tcl_platform(platform) == "windows"} {
+#  package require registry 1.1
+#}
 
 ### Default Configuration
 
@@ -103,7 +103,7 @@ set version "3.0"
 
 if {$tcl_platform(platform) == "unix"} {
    if {[lsearch -exact [array names env] WNHOME] == -1} {
-	set resourcedir "/usr/local/WordNet-3.0/lib/wnres"
+	set resourcedir "/usr/lib/wnres"
    } else {
 	set resourcedir "$env(WNHOME)/lib/wnres"
    }
@@ -114,20 +114,16 @@ if {$tcl_platform(platform) == "unix"} {
    set printcommand "lpr"
 }
 if {$tcl_platform(platform) == "windows"} {
-    set UserRegPath "HKEY_CURRENT_USER\\Software\\WordNet\\3.0"
-    if { [registry values $UserRegPath "wnrc" ] != "" } {
-	eval [ registry get $UserRegPath "wnrc" ]
-    }
-    if { [registry values $UserRegPath "WNHome"] != "" } {
-	 append resourcedir [registry get $UserRegPath "WNHome"] "/lib/wnres"
-    } else {
-    	 set MachineRegPath "HKEY_LOCAL_MACHINE\\Software\\WordNet\\3.0"
-    	 if { [registry values $MachineRegPath "WNHome" ] != "" } {
-     	    append resourcedir [registry get $MachineRegPath "WNHome"] "/lib/wnres"
-	 } else {
-          set resourcedir "C:/Program Files/WordNet/3.0/lib/wnres"
-       }
-    }
+   if {[lsearch -exact [array names env] WNHOME] == -1} {
+	set resourcedir "/usr/lib/wnres"
+   } else {
+	set resourcedir "$env(WNHOME)/lib/wnres"
+   }
+   set configfile "$env(HOME)/.wnrc"
+   if [ file exists $configfile ] {
+       source $configfile
+   }
+   set printcommand "lpr"
 }
 
 ### Startup
@@ -502,18 +498,17 @@ proc saveoptions { } {
 	puts $c "set fontsize $fontsize"
 	puts $c "set fontsizelist \{ $fontsizelist \}"
     } elseif {$tcl_platform(platform) == "windows"} {
-	set c ""
-	append c "set showcontextualhelp $showcontextualhelp\n"
-	append c "set showglosses $showglosses\n"
-	append c "set wordwrap $wordwrap\n"
-	append c "set showfileinfo $showfileinfo\n"
-	append c "set showbyteoffset $showbyteoffset\n"
-	append c "set showsenseflag $showsenseflag\n"
-	append c "set maxhistorylength $maxhistorylength\n"
-	append c "set fontname $fontname\n"
-	append c "set fontsize $fontsize\n"
-	append c "set fontsizelist \{ $fontsizelist \}\n"
-	registry set $UserRegPath "wnrc" $c
+	set c [open $configfile w]
+	puts $c "set showcontextualhelp $showcontextualhelp"
+	puts $c "set showglosses $showglosses"
+	puts $c "set wordwrap $wordwrap"
+	puts $c "set showfileinfo $showfileinfo"
+	puts $c "set showbyteoffset $showbyteoffset"
+	puts $c "set showsenseflag $showsenseflag"
+	puts $c "set maxhistorylength $maxhistorylength"
+	puts $c "set fontname $fontname"
+	puts $c "set fontsize $fontsize"
+	puts $c "set fontsizelist \{ $fontsizelist \}"
     }
 }
  
